core/src/org/pentaho/platform/util/beans/BeanUtil.java

```diff
@@ -103,8 +103,12 @@ public Object getValue( String propertyName ) throws IllegalAccessException, Inv
    * @return <code>true</code> if the bean property can be written to
    */
   public boolean isWriteable( String propertyName ) {
-    return propUtil.isWriteable( bean, propertyName )
-        || ( propUtil.getResolver().isIndexed( propertyName ) && propUtil.isReadable( bean, propertyName ) );
+    try {
+      return propUtil.isWriteable( bean, propertyName )
+          || ( propUtil.getResolver().isIndexed( propertyName ) && propUtil.isReadable( bean, propertyName ) );
+    } catch ( IllegalArgumentException exception ) {
+      return false;
```

Comment:
I think there should be some logging here - at least debug-level logging. This is swallowing the exception completely.